### DISCUSSION TITLE
Need to customize Hubs client Admin panel

### DISCUSSION THREAD

**Discussion by big-javascript-fan** [mozilla/hubs#3267] [General Questions & Discussion] [answered]
I need to customize hubs client admin panel as I can customize hubs client.
Now, I can not run hubs client admin panel on my local end so that I can not see result of customized hubs client admin panel.
Are there any way to do this?

**Comment by robinkwilson** [comment, marked as answer]
Unfortunately without a connection to the db on hubs cloud, you can't run the admin panel locally. The way I've done work on the admin panel on my HC instance is deploy the client to your instance and test there.

I've removed the `npm ci` step to make the deploy process faster: https://gist.github.com/robin-k-wilson/9bc65f2f8065b5dd650966236b43d2f3

Not ideal, made an issue to follow: https://github.com/mozilla/hubs-cloud/issues/123
